src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -213,6 +213,37 @@ public inherited sharing class RD2_OpportunityEvaluationService {
         } set;
     }
 
+    /**
+    * @description Contains all mandatory Recurring Donation fields required for the Opportunity Evaluation Service
+    */
+    private Set<String> rdFields = new Set<String> {
```

Comment:
@andrewyu-salesforce :  if not already done so, can you please verify changes work on a multi-currency org and RD1/RD2 tests execute successfully?
@ravensisco , can you please add multi-currency tests to the test charter?

Reply:
Good catch. I update the code to include CurrencyIsoCode when the org is multi-currency